src/usr/local/www/firewall_rules_edit.php

```diff
@@ -1570,12 +1583,25 @@ function build_flag_table() {
 ))->setHelp('A packet matching this rule can be marked and this mark used to match '.
 	'on other NAT/filter rules. It is called %1$sPolicy filtering%2$s.', '<b>', '</b>');
 
-$section->addInput(new Form_Input(
-	'tagged',
-	'Tagged',
-	'text',
-	$pconfig['tagged']
-))->setHelp('A packet can be matched on a mark placed before on another rule.');
+$group = new Form_Group('Tagged');
+
+$group->add(new Form_Checkbox(
+	'nottagged',
+	'nottagged',
+	'Invert',
```

Comment:
This should be `Invert match`

Reply:
done,
but now GUI is not so pretty

Reply:
I see that now, it forces a word wrap. I guess it's OK to move it back to just 'Invert' and adjust the text below to match.

Reply:
done